core/src/operations/decorators/advice.rs

```diff
@@ -11,6 +11,13 @@ pub enum AdviceInjector {
     /// - root of the tree, 4 elements
     MerkleNode,
 
+    /// Merges two Merkle tree roots into a new root. The merged Merkle root will be pushed onto
+    /// the head of the advice stack.
```

Comment:
As mentioned in the previous comments, we don't need to push anything onto the advice stack.

Reply:
Fixed